ngfizzy/ps-notes
# User Authorization

### WHAT THIS PR DOES

This PR implements authorization of a user. After merging this pr, a user will not be able to assess the dashboard anymore except they are properly logged in.

### How DO WE TEST THAT IT'S WORKING?

* Start the app and try to navigate to  `/dashboard` if you're not authenticated, you should be redirected to be authenticated again.

### ANY OTHER INFORMATION?

The mechanism used for authorization might change later when the API we are consuming changes.
